enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1676,7 +1684,8 @@ func (p *PebbleCache) deleteMetadataOnly(ctx context.Context, key filestore.Pebb
 	if err := db.Delete(fileMetadataKey, pebble.NoSync); err != nil {
 		return err
 	}
-	p.sendSizeUpdate(fileMetadata.GetFileRecord().GetIsolation().GetPartitionId(), key.CacheType(), -1*fileMetadata.GetStoredSizeBytes())
+	mdSize := int64(proto.Size(fileMetadata))
```

Comment:
The old code is not necessarily wrong. deleteMetadataOnly is called if metadata entry is present but the file on disk is missing in which case not counting StoredSizeBytes could lead to incorrect accounting. But same can be said in the other direction. Sticking to the same calculation as the rest of the code might make things simpler.

Reply:
I moved the calculation in the sendSizeUpdate.